app/ante.go

```diff
@@ -31,11 +30,13 @@ func NewAnteHandler(
 ) sdk.AnteHandler {
 	mempoolFeeOptions := txfeestypes.NewMempoolFeeOptions(appOpts)
 	mempoolFeeDecorator := txfeeskeeper.NewMempoolFeeDecorator(*txFeesKeeper, mempoolFeeOptions)
+
 	return sdk.ChainAnteDecorators(
 		ante.NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
 		wasmkeeper.NewLimitSimulationGasDecorator(wasmConfig.SimulationGasLimit),
 		wasmkeeper.NewCountTXDecorator(txCounterStoreKey),
 		ante.NewRejectExtensionOptionsDecorator(),
+		v8.MsgFilterDecorator{},
```

Comment:
Is there a reason this is directly initialized here instead of through a NewMsgFilterDecorator() function as with the mempoolFeeDecorator below?

I understand that in this case it doesn't actually make a difference since the struct itself is empty and doesn't require any inputs to create, but I wonder if breaking this layer of abstraction introduces any risks or just generally makes this decorator a lot less useful/potentially more confusing in case we need to make changes to it in the future?

Reply:
Its cleaner to use the function. Want to make a commit here?

Reply:
Yep!

Reply:
Created new issue [here](https://github.com/osmosis-labs/osmosis/issues/1487#issue-1236190344) to address this in a separate PR after v8 upgrade to avoid an addition round of code reviews. Resolving this discussion for now to remove clutter from this PR